doc: configure ejected create-react-app to support decorators (#5682)
microsoft/fast: packages/web-components/fast-foundation/docs/integrations/react.md

- Use an intermediary like [react-app-rewired](https://www.npmjs.com/package/react-app-rewired)

You can read more about decorator configuration issues [here.](https://github.com/microsoft/fast/issues/4503)
### Configure ejected create-react-app

Eject create-react-app
```shell
npm run eject
```

Install babel plugins
```shell
npm i --save-dev @babel/plugin-proposal-decorators @babel/preset-env
```
#### Configure babel-loader options

Go to `webpack.config.js` file under the `config` folder and find where `babel-loader` is (around line 407).

Add `@babel/preset-env` to presets, this allows you to use latest JavaScript features.

> Targeting specific browser versions prevents Babel to transpile down too much to support old JavaScript versions and increasing file size.

Reviewer: ```suggestion
Targeting specific browser versions prevents Babel from transpiling too much to support old JavaScript versions, increasing file size.
```
